pkg/cmd/create/root.go

```diff
@@ -47,7 +47,7 @@ func init() {
 	// cluster related flags
 	CreateCmd.PersistentFlags().BoolVar(&recreateCluster, "recreate", false, "Delete cluster first if it already exists.")
 	CreateCmd.PersistentFlags().StringVar(&buildName, "build-name", "localdev", "Name for build (Prefix for kind cluster name, pod names, etc).")
-	CreateCmd.PersistentFlags().StringVar(&kubeVersion, "kube-version", "v1.29.2", "Version of the kind kubernetes cluster to create.")
+	CreateCmd.PersistentFlags().StringVar(&kubeVersion, "kube-version", "v1.30.0", "Version of the kind kubernetes cluster to create.")
```

Comment:
Why not latest patch release? 1.30.4?

Reply:
Yeah good idea. I looked at the kind version we are using and its default k8s version (`v1.30.0`).  Bumping it up to the next patch release is a good idea. Your PR would be awesome here 😃